man/oci-image-tool-validate.1.md

```diff
@@ -19,15 +19,15 @@ oci-image-tool validate \- Validate one or more image files
   The reference to validate (should point to a manifest).
   Can be specified multiple times to validate multiple references.
   `NAME` must be present in the `refs` subdirectory of the image.
-  Only applicable if type is image or imageLayout.
+  Only applicable if type is image.
 
 **--type**=""
-  Type of the file to validate. If unset, oci-image-tool will try to auto-detect the type. One of "imageLayout,image,imageZip,manifest,imageIndex,config"
+  Type of the file to validate. One of "image,manifest,imageIndex,config"
```

Comment:
An `image` isn't the layout. The layout is the collection of resources that can make up an image. For example, I can still have an image and not have a layout.

Reply:
Yeah, agree.  And as you said, layout makes up an image. If we have a layout then we have an image. The point is image may be stored in different file types.
But if you don't have a layout, then it will not be a valid OCI image. 
So, how do you think we should change the help info?